inst/builtin/bs5/shiny/_variables.scss

```diff
@@ -65,23 +67,29 @@ $h1-font-size: 2.1rem !default;
 $body-font-weight: 400 !default;
 $headings-font-weight: 400 !default;
 
-$body-color: $gray-900 !default;
-$body-bg: #ffffff !default;
-
+// Spacing
 $hr-margin-y: 2rem !default;
 
+// Buttons
 $input-btn-font-size: 15px !default;
-
 $btn-font-weight: 500 !default;
 $btn-padding-y: 0.75em !default;
 $btn-padding-x: 2em !default;
 
-
+// Form controls
 $form-check-input-width: 1.2em !default;
 $form-check-input-border-radius: 2px !default;
 $form-check-margin-bottom: 0.25rem !default;
 $form-text-color: $red !default;
 
+// Borders
 $input-border-color: $gray-500 !default;
-
 $border-radius: 3px !default;
+
+// Components
+$component-active-bg: $primary !default;
+$component-active-color: color-contrast($component-active-bg, $white) !default;
```

Comment:
Are these defined here so that you can use them below? If so, lets move the `$selectize-*` variables to another file called something like `_mixins.scss` and feed that into the `mixins` slot of a `sass_layer()`

Reply:
Okay, I added support for `_mixins.scss` in `builtin_bundle()` in a way that we could easily extend to support the rest of the `sass_layer()` arguments, and I moved these selectize rules there (and removed the `$component-` rules). 92bf806d6d9a64e6bb5f7c6edefe0d4f10640808

Reply:
FYI, I found the current defaults and I think they're reasonable: https://github.com/rstudio/shiny/blob/1b7709411beeb345af54d159c0af6528fd542525/inst/www/shared/selectize/scss/selectize.bootstrap4.scss#L30-L31

I'm going to leave this as a shiny preset modification.